svm/src/account_loader.rs

```diff
@@ -58,17 +58,37 @@ pub(crate) enum TransactionLoadResult {
 }
 
 #[derive(PartialEq, Eq, Debug, Clone)]
-#[cfg_attr(feature = "dev-context-only-utils", derive(Default))]
+#[cfg_attr(
+    feature = "svm-internal",
+    field_qualifiers(nonce(pub), lamports_per_signature(pub),)
```

Comment:
why do these now need to be `pub`? should `compute_budget_limits` also be `pub` for internals?

Reply:
This is used by some SVM integration tests. Earlier it did not need access to it. `compute_budget_limits` could also be made pub, it's not necessary.

Reply:
https://github.com/anza-xyz/agave/pull/4813/files#diff-969a5f1fd29bf2de600c91bde61a7be4b4655499a6b73863ba84fcbfda535b67R406